### PR TITLE
Add in User Agent Randomizer / Anonymization

### PR DESCRIPTION
**Pull request by labboy0276**
Adds User Agent Randomizer / Anonymization, helps with headless users.

### CODE REVIEW

**Comment by samc621** [comment]
Hi @labboy0276 I am testing this now. It appears to help with some issues on the footsites, but not doing anything for the CORS issue on Nike. It triggers this modal.

Screen Shot 2021-07-09 at 11 22 35 AM

 I think you mentioned hitting that issue, correct? Was this fix helping you bypass it?

**Comment by labboy0276** [comment]
@samc621 Yeah I don't think it will do anything for a CORS issue.  It just mocks the user agent for us.  

If you want to put the output of the CORS error into #28 I will see if I can figure it out this weekend or next.

**Comment by samc621** [comment]
@labboy0276 yup I gotcha, randomizing the UA is a good idea anyways. Merging this now!

As for the Nike error, I saw this modal when I ran the bot in headful mode. I didn't realize it was a CORS issue until I checked the requests in the network tab. I don't think there was anything in the response body, just a CORS error. You should be able to repro it pretty easily.

File: helpers/cluster.js [hunk 1]
```diff
@@ -14,7 +16,11 @@ const { storePageInTaskCache } = require('./task-cache');
 
 const sites = require('../sites');
 
+const userAgent = new UserAgent();
+
 puppeteer.use(StealthPlugin());
+puppeteer.use(Ua());
+
```
Comment (samc621) [inline helpers/cluster.js:23]: Actually @labboy0276 it looks like this extra line break is causing an error with the linter. Mind removing it? Then this LGTM!

**Review by samc621 — CHANGES_REQUESTED** [review]
@labboy0276 thanks for this, and congrats on your first PR! Just one inline issue.

**Comment by samc621** [comment]
@labboy0276 just pinging you again in case you missed my [comment](https://github.com/samc621/SneakerBot/pull/39#discussion_r667214467) above.

**Comment by labboy0276** [comment]
@samc621 all set

**Review by samc621 — APPROVED** [review]
@labboy0276 thank you for this, and congrats on your first PR! Your work on the antibot stuff will be very helpful and appreciated!

**Pull request merged**